src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/DefaultGATKReadFilterArgumentCollection.java

```diff
@@ -23,6 +24,11 @@ public class DefaultGATKReadFilterArgumentCollection extends GATKReadFilterArgum
             doc="Read filters to be applied before analysis", optional=true, common = true)
     public final List<String> userEnabledReadFilterNames = new ArrayList<>(); // preserve order
 
+    @Argument(fullName = ReadFilterArgumentDefinitions.INVERTED_READ_FILTER_LONG_NAME,
```

Comment:
Ah, I thought you were going to implement special handling of `not` in the read filter name which sounded complicated.  This is much clearer.

Reply:
I considered it but when we add a read filter named Not___ReadFilter i really didn't want to deal with NotNot___ReadFilters cropping up...

Reply:
We could potentially allow `!ReadFilter` because `!` shouldn't be allowed in read filter names, but yeah, it would be more complicated I think.